audobject/core/api.py

```diff
@@ -72,6 +75,12 @@ def from_dict(
         override_args,
     )
 
+    if isinstance(object, Object):
```

Comment:
I guess this covers the theoretical case that we load something from YAML that was not stored as an `audobject.Object`, but we were still able to load it?

Reply:
Could you add a test for this?

Reply:
> I guess this covers the theoretical case that we load something from YAML that was not stored as an audobject.Object, but we were still able to load it?

Yes, you can also initialize an object from YAML that does not derive from `audobject.Object`, e.g.:

```python
yaml_s = r'''
$audbackend.core.filesystem.FileSystem:
  host: ~/host
  repository: repo
'''

audobject.from_yaml_s(yaml_s)
```
```
<audbackend.core.filesystem.FileSystem object at 0x7efe1b0ecbe0>
```

> Could you add a test for this?

It's already covered by `tests/test_install.py` where we initialize `artifactory.ArtifactoryPath`, since it does not allow to change `__init__` and raised an error until I added the check that `object` must derive from `audobject.Object`.

Reply:
Ah, great, I was already wondering why you thought about the `if`-statement without an failing example ;)